pkgs/applications/audio/dragonfly-reverb/default.nix

```diff
@@ -5,8 +5,8 @@ stdenv.mkDerivation rec {
 
   src = fetchgit {
```

Comment:
Would you mind changing it to `fetchFromGitHub`?

Reply:
This:
```
  src = fetchFromGitHub {
    owner = "michaelwillis";
    repo = "dragonfly-reverb";
    rev = "0.9.3";
    sha256 = "0a1zj58bgfg5gpzn50bywv3hzh8rfign7gazi2r6wrn9ky71xyb7";
  };
``` 
doesn't build:
```
make[1]: Leaving directory '/build/source/freeverb3'
make -C dpf/dgl
make[1]: *** dpf/dgl: No such file or directory.  Stop.
make: *** [Makefile:15: libs] Error 2
```

I create [an issue](https://github.com/michaelwillis/dragonfly-reverb/issues/29).

Reply:
Oh I see. The sub folder even dpf points to another repository.

Reply:
You would need to set `fetchSubmodules` to true like:

```nix
  src = fetchFromGitHub {
    owner = "michaelwillis";
    repo = "dragonfly-reverb";
    rev = "0.9.3";
    sha256 = "<hash>";
    fetchSubmodules = true;
  };
```

SHA hash might need to be recomputed